.mk/k8s.mk

```diff
@@ -331,13 +331,13 @@ k8s-crossconnect-monitor-save: ${CONTAINER_BUILD_PREFIX}-crossconnect-monitor-sa
 k8s-crossconnect-monitor-load-images:  k8s-start $(addsuffix -load-images,$(addprefix ${CLUSTER_RULES_PREFIX}-,crossconnect-monitor))
 
 .PHONY: k8s-admission-webhook-build
-k8s-admission-webhook-build: ${CONTAINER_BUILD_PREFIX}-admission-webhook-build
+k8s-admission-webhook-build: ${CONTAINER_BUILD_PREFIX}-admission-webhook-build ${CONTAINER_BUILD_PREFIX}-nsm-init-build
 
 .PHONY: k8s-admission-webhook-save
-k8s-admission-webhook-save: ${CONTAINER_BUILD_PREFIX}-admission-webhook-save
+k8s-admission-webhook-save: ${CONTAINER_BUILD_PREFIX}-admission-webhook-save ${CONTAINER_BUILD_PREFIX}-nsm-init-build
```

Comment:
Should be `${CONTAINER_BUILD_PREFIX}-nsm-init-build` --> `${CONTAINER_BUILD_PREFIX}-nsm-init-save`?

Reply:
Of course